src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -472,6 +475,20 @@ namespace AppInstaller::Runtime
             result = GetPathDetailsForPackagedContext(PathName::LocalState, forDisplay);
             result.Path /= s_CheckpointsDirectory;
             break;
+        case PathName::CLIExecutable:
+            result.Path = GetKnownFolderPath(FOLDERID_LocalAppData);
+            result.Path /= s_WindowsApps_Base;
+            if (IsReleaseBuild())

```

Comment:
This is not the right thing to key on here.  We have already overloaded `USE_PROD_CLSIDS` to cover targeting the dev vs prod package, so I'm fine with using that here.

Additionally, you should use the package family name in the path to ensure that it is going to the correct alias.  For instance, `%LOCALAPPDATA%\Microsoft\WindowsApps\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\winget.exe`